lang/strings_english.txt

```diff
@@ -1259,6 +1259,8 @@ $s_manage_filter_page_title = 'Manage Filters';
 $s_manage_filter_edit_page_title = 'Edit filter';
 $s_delete_filter_button = 'Delete';
 $s_apply_filter_button = 'Apply';
+$s_temporary_filter = 'Temporary filter';
+$s_set_as_persistent_filter = 'Set as persistent filter';
```

Comment:
Maybe a native speaker can comment.
Shouldn't it be `Set as persisted Filter``

Reply:
I think _persistent_ is correct in this context. I would say _**Save** as persistent filter_ though.